apps/src/JsDebuggerUi.js

```diff
@@ -410,7 +434,7 @@ JsDebuggerUi.prototype.onPauseContinueButton = function() {
 };
 
 JsDebuggerUi.prototype.updatePauseUiState = function() {
```

Comment:
There's loads of potential to refactor and clean up all of the debug control state stuff - ideally, into something closer to a state machine.  I didn't want to do that part in this PR though.